src/components/leaflet/LeafletMap.jsx

```diff
@@ -13,12 +14,13 @@ const customIcon = new Icon({
 const LeafletMap = (props) => {
   return (
     <MapContainer
-      key={`${props.coordinates[0]}`}
+      // key={`${props.coordinates[0]}`}
```

Comment:
It has to be removed not just commented out.

Reply:
done